frontroomsbot/bot.py

```diff
@@ -48,6 +48,15 @@ async def flip(interaction: discord.Interaction):
     await interaction.response.send_message(f"{result}")
 
 
+@tree.command(name="kasparek", description="Zjistí jakého máš kašpara", guild=guild)
+async def kasparek(interaction: discord.Interaction):
+    unit = choice(["m", "cm", "mm", "nm"])
+    result = round(uniform(0, 50), 5)
```

Comment:
not a fan of floats for this and especially not 5 digit floats

Reply:
I feel like having at least 2 digit float so we have fewer exact number could be a good thing.